src/components/analyses/details/ParamsPanel.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * @author sriram
+ *
+ * A panel that displays analysis params.
+ *
+ *
+ */
+
+import React, { useState } from "react";
+import { injectIntl } from "react-intl";
+import { parseNameFromPath } from "../../data/utils";
+import DEErrorDialog from "../../utils/error/DEErrorDialog";
+import ErrorTypography from "../../utils/error/ErrorTypography";
+import TableLoading from "../../utils/TableLoading";
+import ids from "../ids";
+import intlData from "../messages";
+import ArgumentTypes from "./ArgumentTypes";
+import { EnhancedTableHead, formatMessage, withI18N } from "@cyverse-de/ui-lib";
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableRow,
+    Typography,
+} from "@material-ui/core";
+
+function ParameterValue(props) {
+    const {
+        parameter: {
+            info_type,
+            param_type,
+            param_value: { value },
+        },
+    } = props;
+
+    let valid_info_type =
+        !(info_type === ArgumentTypes.PARAM_TYPE.REFERENCE_GENOME) &&
+        !(info_type === ArgumentTypes.PARAM_TYPE.REFERENCE_SEQUENCE) &&
+        !(info_type === ArgumentTypes.PARAM_TYPE.REFERENCE_ANNOTATION);
```

Comment:
I think `ArgumentTypeUtils` functions can be used in this file:
```suggestion
        !isReferenceGenomeType(info_type);
```